src/slack_message_formatter.ts

```diff
@@ -27,12 +17,23 @@ export function formatMessageHistory(
   const formattedMessages = messageObjects.map((messageObject) => {
     const timeSinceEpochSecs = Date.parse(messageObject.timestamp) / 1000;
     // See https://api.slack.com/reference/surfaces/formatting#visual-styles
-    const specialSlackTimestamp = `*(<!date^${timeSinceEpochSecs}^{date_num} {time_secs}|${messageObject.timestamp}>)*`;
-    const messageSender = `*${getMessageSender(messageObject, userId)}*`;
-    return [specialSlackTimestamp, messageSender, messageObject.message].join(
-      ' '
-    );
+    const specialSlackTimestamp = `<!date^${timeSinceEpochSecs}^{time} {date_short}|${messageObject.timestamp}>`;
+    if (messageObject.direction == 'INBOUND') {
+      return (
+        `:bust_in_silhouette: *Voter ${userId}*  ` +
+        specialSlackTimestamp +
+        '\n' +
+        formatMessageBlock(messageObject.message, '*')
+      );
+    } else {
```

Comment:
I love the gear icon and the bust_in_silhouette icons! One other thing to keep in mind is that voters can be routed manually to a new channel, in which case the conversation history is similarly carried over to the new channel, including messages from other volunteers. Can we have a separate icon for volunteer messages? Maybe a different bust_in_silhouette color?

Here is an example: https://voteamericahelpline.slack.com/archives/G01BQE0THSA/p1602363906001000

You can test this locally by going to the `#admin-control-room` and typing `@${yourbotuser} ROUTE_VOTER ${userid} ${phonenumber with leading "+1"} ${destinationSlackChannel without leading #}`. Also, please also test routing a voter to a new channel, having additional conversation in that channel, and then routing them back to their original channel, which should post in their original thread in the original channel, adding only the incremental message history that was missed.